include/wpe/keysyms.h

```diff
@@ -5,7 +5,7 @@
 #ifndef wpe_keysyms_h
 #define wpe_keysyms_h
 
-/* This file is autogenerated; use https://github.com/WebPlatformForEmbedded/WPEBackend/blob/master/scripts/update-keysyms-header to update it. */
+/* This file is autogenerated; use https://github.com/WebPlatformForEmbedded/libwpe/blob/master/scripts/update-keysyms-header to update it. */
```

Comment:
Now you have to rename the git repo too!

Reply:
Yes, that's the plan :wink: 

GitHub will make a `WPEBackend` → `libwpe` redirection automatically.